src/SonarScanner.MSBuild.PreProcessor/Interfaces/IPreprocessorObjectFactory.cs

```diff
@@ -32,7 +32,7 @@ public interface IPreprocessorObjectFactory
         /// </summary>
         /// <param name="args">Validated arguments.</param>
```

Comment:
The documentation is not up to date. One parameter is added and the others are missing. I think we should either add the remaining ones or remove the `args` too.

Reply:
Removed, as I do not see much value to this